tokens.xml

```diff
@@ -8287,14 +8374,18 @@ This creature can't be enchanted.</text>
             <reverse-related count="x">Gadrak, the Crown-Scourge</reverse-related>
             <reverse-related>Gala Greeters</reverse-related>
             <reverse-related>Galazeth Prismari</reverse-related>
+            <reverse-related count="1">Ganax, Astral Hunter</reverse-related>
```

Comment:
I have changed this so that it defaults to 1, and have a count=x as an exclude so users have the option to create more than 1 when needed, and so it doesn't show up for most uses.